tests/fixtures/phpcs/DrupalClassCorrect.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\testmodule;
+
+use Drupal\BlockBase;
+
+/**
+ * Some service.
```

Comment:
If it where a service, it would be named as such

Reply:
Fixed